package/MDAnalysis/coordinates/base.py

```diff
@@ -1023,6 +1023,12 @@ def timeseries(self, asel: Optional['AtomGroup']=None,
 
         .. versionadded:: 2.4.0
         """
+        if asel:
```

Comment:
Same as above, we should support both keyword arguments and docs should reflect this.